tests/unit/s2n_tls13_support_test.c

```diff
@@ -27,11 +27,18 @@
 #include "tls/s2n_connection.h"
 #include "tls/s2n_cipher_suites.h"
 
+#include "crypto/s2n_fips.h"
+
 int main(int argc, char **argv)
 {
     BEGIN_TEST();
     EXPECT_SUCCESS(s2n_disable_tls13_in_test());
 
+    if (s2n_is_in_fips_mode()) {
+        /* Skip when FIPS mode is set as FIPS mode does not support TLS1.3 */
+        END_TEST();
+    }
```

Comment:
Is this actually true? At the moment, this might just be an artifact of only Openssl-1.0.2 supporting FIPs, and even then are you sure we couldn't do TLS1.3 in some specific cases?

Also, AWS-LC plans to (already does?) support fips + TLS1.3. In that case, what config should we be using?

Reply:
Hmm that's a good point. It seems like if you're using FIPS, we're never going to need the other two configs. But if Openssl FIPS currently doesn't support TLS13, is there a way to make these tests pass for all libcrypto?

Reply:
That's a behavior change. At the moment, if you're using FIPS+TLS1.3, you'd get the TLS1.3 config.